lib/js/components/Layout/Layout.vue

```diff
@@ -0,0 +1,261 @@
+<template>
+	<div class="layout">
+		<div v-if="sideMenuVisible" class="layout__navigation">
+			<div class="layout__mainMenu">
+				<slot name="mainMenu" />
+			</div>
+			<div class="layout__sideNav">
+				<slot name="sideNav" />
+			</div>
+		</div>
+		<div
+			class="layout__contentColumn"
+			:class="{
+				'-fullWidth': isContentFullWidth,
+				'-max900': !isContentFullWidth,
```

Comment:
similar to https://github.com/bethinkpl/design-system/pull/295/files#r1193359026
don't use pixel value in class name